### DISCUSSION TITLE
Is there anything I can do to fix buffering?

### DISCUSSION THREAD

**Discussion by hyperglass** [iptv-org/community#149] [Q&A] [answered]
channels buffering every 5 seconds, I am using Tivimate. I have 50mbps Internet connection.

**Comment by BellezaEmporium** [comment, marked as answer]
Please precise the channels you're trying to watch, from which country, and even if the channels buffer, it might be either an unstable connection on your side, or server bandwidth being too low.

**Reply by hyperglass** [reply]
Dhinchak TV (540p) ; Country : India.
